docs/monster-ui/getJsoneditor().md

```diff
@@ -0,0 +1,29 @@
+title: getJsoneditor()
+
+# monster.ui.getJsoneditor()
+The `monster.ui.getJsoneditor()` method gets a JSONEditor instance from the DOM with the editor's container
```

Comment:
No need for a tldr when the helper is this simple.

Also, please don't forget to end your sentences with periods.
```suggestion
```